core/server/api/canary/users.js

```diff
@@ -121,15 +122,19 @@ module.exports = {
             }
         },
         permissions: true,
-        query(frame) {
+        async query(frame) {
+            const backupPath = await dbBackup.backup();
+            const parsedFileName = path.parse(backupPath);
+            const filename = `${parsedFileName.name}${parsedFileName.ext}`;
```

Comment:
@kevinansfield was wondering what are your thoughts on passing just the filename from the response and similarly only accepting the filename VS returning a whole path. 

Applied the former approach as it is safer compared to exposing internal file structure. The only downside of this approach is if somebody changes the `config.get('paths').contentPath` after fetching the file, it no longer will be returned. But this is an acceptable side effect I think.

Reply:
Agreed we should only be exposing a filename and not any path structure 👍